doc/man7/EVP_MAC-HMAC.pod

```diff
@@ -43,27 +43,27 @@ Sets the name of the underlying digest to be used.
 =item "properties" (B<OSSL_MAC_PARAM_PROPERTIES>) <UTF8 string>
 
 Sets the properties to be queried when trying to fetch the underlying digest.
-This must be given together with the digest naming parameter to be considered
-valid.
+This must be given together with the digest naming parameter ("digest", or
+B<OSSL_MAC_PARAM_DIGEST>) to be considered valid.
 
 =item "digest-noinit" (B<OSSL_MAC_PARAM_DIGEST_NOINIT>) <integer>
 
-A simple flag to set the MAC digest to not initialise the
-implementation specific data. The value 0 or 1 is expected.
+A flag to set the MAC digest to not initialise the implementation
+specific data.
+The value 0 or 1 is expected.
 
 =item "digest-oneshot" (B<OSSL_MAC_PARAM_DIGEST_ONESHOT>) <integer>
 
-A simple flag to set the MAC digest to be a oneshot operation.
+A flag to set the MAC digest to be a oneshot operation.
```

Comment:
probably oneshot is 2 words..

Reply:
A quick google shows me that "one-shot" seems to be common.  I'll got for that